scripts/troubleshoot.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+dkms_name="razer_chroma_drivser"  # TODO Rename when dkms module is renamed.
+
+echo "This script will now try to determine why the razer driver isn't working correctly."
+
+# Step 1 - check if there is a razer device connected.
+lsusb=$(lsusb | grep '1532:')
+if [ -z "$lsusb" ]; then
+  echo "ERROR: Linux doesn't see any Razer device connected. Make sure it is connected! Exiting."
+  exit 1
+fi
+
+# Step 2 - check if a module is already loaded.
+lsmod=$(lsmod | grep razer)
+if [ -z "$lsmod" ]; then
+  echo "INFO: There is no kernel module loaded."
+fi
+
+# Step 3 - check if the module can load correctly
+echo "Please enter your 'sudo' password for priviliged operations."
+module_name="razerkbd"
```

Comment:
I actually don't want the whole script to be run as root so I think this should work just fine...